### PR TITLE
US01-04 Carousel 

### PR DESCRIPTION
**Pull request by Wowlfy**
carousel usable for any page

### CODE REVIEW

**Review by ChristianOlmedo — DISMISSED** [review]

File: src/View/User/Includes/carousel.html.twig [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  </div>
+  <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide="prev">
+    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+    <span class="visually-hidden">Previous</span>
+  </button>
+  <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide="next">
+    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+    <span class="visually-hidden">Next</span>
+  </button>
+</div>
```
Comment (LucyMerlier) [inline src/View/User/Includes/carousel.html.twig]: 2 spaces for indentation => should be 4

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
indentation, see comment

File: src/View/User/Includes/carousel.html.twig (hunk 1 above)
Comment (Wowlfy) [reply src/View/User/Includes/carousel.html.twig]: Ty

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
"carousel.html.twig" should start with an underscore

**Review by ChristianOlmedo — DISMISSED** [review]
The indentation problem in _carousel.html.twig must be due to github but it's ok

**Review by LucyMerlier — DISMISSED** [review]

**Review by Loic-Code — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**